[IMP] sales: update simplified pricelist settings

dikd-odoo opened the pull request:
Updating Pricelists picture and documentation text for 17.4+, simplified Pricelists description in settings, removed 'Pricing Strategy Options' section. Also cleaning up language in documentation.

+Added updates:
-Revised "Pricelist configuration" section to match current UI flow
-Removed Multiple & Advanced Pricelists sections as they are deprecated in 17.4+
-Updated images in "Pricelists configuration" and "Price Rules tab" sections to match current UI and style guide
-Updated language for clarity and concision
-Began staging content changes in intro to re-format this into a parent doc for Sales pricelists

Task: https://www.odoo.com/odoo/my-tasks/4343498

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/11709.png)](https://mergebot.odoo.com/odoo/documentation/pull/11709)

dikd-odoo commented:
Hi @larm-odoo this PR is ready for peer review, when you have a chance. Thank you!

larm-odoo commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -10,35 +10,19 @@ quantity sold, and more) in order to apply certain prices or discounts.
-  product.
-- :guilabel:`Advanced price rules (discounts, formulas)`: provides the option to create detailed
-  price rules and apply discounts, margins, and roundings.
+To enable pricelists in Odoo Sales app, first navigate to :menuselection:`Sales app -->
+Configuration --> Settings`. In the :guilabel:`Pricing` section, tick the checkbox next to the
```
```suggestion
To enable pricelists in the Odoo Sales app, first navigate to :menuselection:`Sales app -->
Configuration --> Settings`. In the :guilabel:`Pricing` section, tick the checkbox next to the
```

larm-odoo replied:
The new standard is to have the app names bolded with double asterisks

larm-odoo commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -10,35 +10,19 @@ quantity sold, and more) in order to apply certain prices or discounts.
-  price rules and apply discounts, margins, and roundings.
+To enable pricelists in Odoo Sales app, first navigate to :menuselection:`Sales app -->
+Configuration --> Settings`. In the :guilabel:`Pricing` section, tick the checkbox next to the
+:guilabel:`Pricelists` feature. Click :guilabel:`Save` to save all changes.
```
```suggestion
:guilabel:`Pricelists` feature, and click :guilabel:`Save` to save all changes.
```
Not necessary!

larm-odoo reviewed (approved):
Hi @dikd-odoo - super simple, easy update! Just a couple of minor suggestions. Approving because they are totally unnecessary ad this is good as-is!

dikd-odoo commented:
Thank you for the suggestions, @larm-odoo! 

Hi @Felicious this PR is ready for final review, when you have a chance. Thank you!

Felicious reviewed (approved):
Your changes look great to me, @dikd-odoo ! Grateful for the effort you’ve put into improving clarity and conciseness in this document. Approving now! 😊

That said, I need to adjust the point value of this PR from **3 points** to **1 point**. I’m  sorry for the confusion, and I take partial responsibility—our current guidelines for task scoring could be clearer, and we're working on a document to better explain these specifications.

Here’s a quick summary of how tasks are scored:

- **3 points**: A significant rework where ~50-80% of the document is revised.
- **2 points**: Version-specific updates or improvements to a single section.
- **1 point**: Tiny edits such rewriting a few sentences.

Your rewrite of lines 16–28 into the streamlined version at lines 16–18 is fantastic—clearer and more concise! However, since this PR focuses on rewording a few sentences rather than updating the entire section or addressing outdated UI comprehensively, it qualifies as a small cosmetic change worth 1 point.

To bring this PR to 2 points, you’d need to update the rest of the section (from the **Pricelist configuration** heading to the **Price rules tab** section), ensure all instructions are fully relevant for saas 17.4, refine the rest for clarity and conciseness, and replace outdated images.

If you'd like to make the changes to be 2 points, tag me again after you've made the updates, and I'll review it again. Let me know if you have any questions!

dikd-odoo commented:
Hi @Felicious thank you for the feedback! I further edited the section "Pricelists configuration," as well as updated the following sections:

-Removed Multiple & Advanced Pricelists sections as they are deprecated in 17.4+
-Updated images in "Pricelists configuration" and "Price Rules tab" sections to match current UI and style guide
-Updated language for clarity and concision
-Began staging content changes in intro to re-format this into a parent doc for Sales pricelists

This is ready for a second look, when you have time. Thank you!

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
 
-To add another product line, click :guilabel:`Add a line` again, and repeat the process. There is no
-limit to how many products can be added in the :guilabel:`Price Rules` tab of a pricelist form.
+.. example::
+   To formulate a 100% markup (or 2 times the cost of the product), with a $5 minimum margin, set
+   the :guilabel:`Based on` field to :guilabel:`Cost`, the :guilabel:`Discount` to `-100`, and the
+   :guilabel:`Margins` to `5`. This is often seen in retail situations.
```
good example!

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
-For more information, check out the following section: :ref:`Multiple prices per product
-<sales/multiple-prices-per-product>`.
+   .. image:: pricing/formula-markup-cost-example.png
+      :align: center
```
```suggestion
```

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
-:guilabel:`Price Rules` tab on pricelist forms provides the option to configure detailed price rules
-based on formulas.
+   .. image:: pricing/formula-discount-example.png
+      :align: center
```
```suggestion
```

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,43 +2,26 @@
-  price rules and apply discounts, margins, and roundings.
+To enable pricelists in the Odoo **Sales** app, first navigate to :menuselection:`Sales app -->
+Configuration --> Settings`. In the :guilabel:`Pricing` section, tick the checkbox next to the
+:guilabel:`Pricelists` feature, and click :guilabel:`Save` to save all changes.
```
The image below is from a previous version (: it looks slightly different in saas-17.4+
![image](https://github.com/user-attachments/assets/84b41f9e-cf4e-4664-8298-cd213709ce3d)

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
-   For example, for two pricelists with different rules, but same criteria (e.g., same website, same
-   country), only the *first* pricelist in the list is applied.
+   If there is no specific pricelist configured on a sales quotation, the :guilabel:`Default`
+   pricelist will be applied.
```
```suggestion
   pricelist is applied.
```
suggestion for brevity. Is this still true?

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
+   In Odoo 17 (and above), you are *not required* to enter a pricelist in the :guilabel:`Pricelist`
+   field on a sales quotation form in order to confirm it (i.e. turn it into a sales order).
+
+   The chatter section is available in Odoo 17 and above on pricelist forms, which allows you to add
```
since we mentioned the version earlier, it can be implied (:
```suggestion
   The chatter section is available on pricelist forms, which allows you to add
```

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
 
-   It should also be noted that, in Odoo 17 (and above), a chatter section can be found on pricelist
-   forms, which enhances the ability to communicate about them.
+Editing and creating pricelists
```
maybe "creating and editing"?

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
+-------------------------------
 
-From the :guilabel:`Pricelists` page, either select the desired pricelist to edit, or click
+From the :guilabel:`Pricelists` page, either select the pricelist you would like to edit, or click
```
```suggestion
From the :guilabel:`Pricelists` page, either select the pricelist to edit, or click
```

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
 configured in a number of different ways.
 
-.. image:: pricing/pricelist-detail-form.png
+.. image:: pricing/18-sales-new-pricelist-form.png
```
and remember to remove "align"!

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
-With the :guilabel:`Multiple prices per product` setting activated, the :guilabel:`Price Rules` tab
-on pricelist forms provides the option to add specific products, with a specific price, to a
-pricelist.
+- :guilabel:`Product`: Select one or more products to which this pricelist will apply.
```
```suggestion
- :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one or more products to which this pricelist will apply.
```

Felicious commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -50,87 +33,104 @@ modified at any time.
 
-For more information, check out the following section: :ref:`Multiple prices per product
-<sales/multiple-prices-per-product>`.
+   .. image:: pricing/formula-markup-cost-example.png
```
Also which version did u take this screenshot from? mine looks a little different, and i was testing in a 17.4 runbot
![image](https://github.com/user-attachments/assets/b602a7db-1e31-4b83-9ba6-3ba75e15d5fe)

Felicious reviewed (commented):
Hi @dikd-odoo ! Good job looking through the rest of the document and deleting sections about depreciated features. I've added a few more comments to consider, and after working through those, feel free to send this along to final review!

dikd-odoo commented:
Thank you for the helpful final review, @Felicious!

Hi @samueljlieber this PR is ready for tech review when you have a chance. Thank you!

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,135 +2,131 @@
+adjust sales prices. This adjustment can apply to specific customers, customer groups, sales orders,
+time periods, etc., and is useful for creating pricing strategies and optimizing sales margins.
 
+Odoo *Sales* has a useful pricelist feature that can be tailored to fit any unique pricing strategy.
 Pricelists suggest certain prices, but they can always be overridden on the sales order.
```
Please use bold formatting for app names :)
```suggestion
Odoo **Sales** has a useful pricelist feature that can be tailored to fit any unique pricing
strategy. Pricelists suggest certain prices, but they can always be overridden on the sales order.
```

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,135 +2,131 @@
-
 After activating and saving the :guilabel:`Pricelists` feature, the :guilabel:`Settings` page
-reloads and, from here, either select the :guilabel:`Pricelists` link (beneath the
+reloads. From here, either select the :guilabel:`Pricelists` link (beneath the
 :guilabel:`Pricelists` feature on the :guilabel:`Settings` page), or navigate to
```
Nitpick suggestion to add the icon when it is part of a button :)
```suggestion
reloads. From here, either select the :icon:`oi-arrow-right` :guilabel:`Pricelists` link (beneath
the :guilabel:`Pricelists` feature on the :guilabel:`Settings` page), or navigate to
```

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,135 +2,131 @@
 
-.. image:: pricing/pricelists-page.png
-   :align: center
+.. image:: pricing/18-sales-pricelist-page.png
    :alt: How the pricelists page looks in Odoo Sales.
```
This image (`18-sales-pricelist-page.png`) is too wide at 1554 px and is uncompressed, please reduce its width to be at or below 933px and please compress with pngquant

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,135 +2,131 @@
 
-.. image:: pricing/pricelist-detail-form.png
-   :align: center
+.. image:: pricing/18-sales-new-pricelist-form.png
    :alt: How the pricelist detail form looks in Odoo Sales.
```
This image (`18-sales-new-pricelist-form.png`) is too wide at 1231 px and is uncompressed, please reduce its width to be at or below 933px and please compress with pngquant

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,135 +2,131 @@
 
-For more information, check out the following section: :ref:`Multiple prices per product
-<sales/multiple-prices-per-product>`.
+   .. image:: pricing/formula-markup-cost-example.png
+      :alt: How it looks to formulate a markup cost with 5 dollar minimum margin in Odoo Sales.
```
This image (`formula-markup-cost-example.png`) is uncompressed, please compress with pngquant :)

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,135 +2,131 @@
-With the :guilabel:`Multiple prices per product` setting activated, the :guilabel:`Price Rules` tab
-on pricelist forms provides the option to add specific products, with a specific price, to a
-pricelist.
+- :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one
+   or more products to which this pricelist will apply.
```
Text in a list **must** align with the first character after the bullet and space. Please remove the extra space here :)
```suggestion
- :guilabel:`Product`: If selected in the :guilabel:`Apply To` field, use this field to choose one
  or more products to which this pricelist will apply.
```

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,135 +2,131 @@
-for which a specific price should be applied.
+  - :guilabel:`Discount`: Enter the percentage to be discounted. A mark-up can be configured by
+    using a negative value in this field.
 
```
Please remove the empty line between the "Discount" and "Formula" bullet points
```suggestion
```

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing.rst:
```diff
@@ -2,135 +2,131 @@
-To configure the price of the product for this specific pricelist, enter the desired amount under
-the :guilabel:`Price` column. Then, there is the option to add a :guilabel:`Start Date` and
-:guilabel:`End Date` to the configured product price, if desired.
+.. note::
+   Rounding is applied *after* the discount and *before* the extra fee.
+
+    - :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount` and
+      :guilabel:`Round off to` have been applied.
+
+  - :guilabel:`Fixed Price`: Enter fixed price for this pricelist. When applied, all product lines
+    in the quotation form will be updated to this price.
```
It appears this note should be nested under the "Round off to" bullet, please indent and fix the note admonition bullet formatting :)
```suggestion
      .. note::
         Rounding is applied *after* the discount and *before* the extra fee.

         - :guilabel:`Extra Fee`: Fixed amount to be added or subtracted once :guilabel:`Discount`
           and :guilabel:`Round off to` have been applied.
         - :guilabel:`Fixed Price`: Enter fixed price for this pricelist. When applied, all product
           lines in the quotation form will be updated to this price.
```

samueljlieber reviewed (changes requested):
Hi @dikd-odoo! Really great work on this doc to remove the deprecated sections and expand with some great examples! I have a small handful of technical changes for you, please see my suggestions and following comments :)

Please run `make review` on both the file (`content/applications/sales/sales/products_prices/prices/pricing.rst`) and the image folder (`content/applications/sales/sales/products_prices/prices/pricing`), there are some early line breaks and uncompressed images ;)

There are also a number of unused images, the following can be removed:
- `pricing/create-pricelist-rules-popup.png`
- `pricing/extra-prices-smartbutton.png`
- `pricing/formula-computation-options.png`
- `pricing/price-rules-product-page.png`
- `pricing/sales-pricelist-formula-computation.png`

I'm requesting changes just so I can double check these images once you have had a chance to address them, thank you for your work! Let me know if you have any Qs!

dikd-odoo commented:
Hi @samueljlieber thank you for the helpful review! I've implemented your changes, removed the unused images, and compressed the images in the folder. This is ready for another look, please let me know if you need anything else. Thank you!

samueljlieber commented on Makefile:
```diff
@@ -96,4 +96,4 @@ review:
 	if [ -z "$$line_length" ]; then line_length=100; fi; \
 	export REVIEW=1; \
-	python tests/main.py --max-line-length=$$line_length $(SOURCE_DIR)/$$path
+	python3 tests/main.py --max-line-length=$$line_length $(SOURCE_DIR)/$$path
```
Make sure not to push up any unintended edits ;)
```suggestion
	python tests/main.py --max-line-length=$$line_length $(SOURCE_DIR)/$$path
```

samueljlieber commented on content/applications/sales/sales/products_prices/prices/pricing/configuration-tab.png:
No need to re-compress these images, we want to avoid unnecessarily modifying images :)

samueljlieber reviewed (approved):
Hi @dikd-odoo, thank you for addressing my changes! I am going to push up a commit that reverts your accidental edit to the Makefile as well as the images that were re-compressed. Thank you for your work, I will merge after I commit and the checks pass.

samueljlieber commented:
@robodoo r+
